ci-operator/step-registry/baremetalds/assisted/operator/bmh/baremetalds-assisted-operator-bmh-commands.sh

```diff
@@ -0,0 +1,197 @@
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  name: {{ infraenv_name }}
+spec:
+  clusterRef:
+    name: {{ cluster_name }}
+    namespace: {{ assisted_namespace }}
+  agentLabelSelector:
+    matchLabels:
+      bla: aaa
+  pullSecretRef:
+    name: {{ pull_secret_name }}
+  proxy:
+    httpProxy: http://11.11.11.33
+    httpsProxy: http://22.22.22.55
+  sshAuthorizedKey: '{{ ssh_public_key }}'
+  ignitionConfigOverride: '{"ignition": {"version": "3.1.0"}, "storage": {"files": [{"path": "/etc/someconfig", "contents": {"source": "data:text/plain;base64,aGVscGltdHJhcHBlZGluYXN3YWdnZXJzcGVj"}}]}}'
+  nmStateConfigLabelSelector:
+    matchLabels:
+      some-user-defined-label-name: some-user-defined-label-value
+EOCR
+
+cat > clusterDeployment.j2 <<EOCR
+apiVersion: hive.openshift.io/v1
+kind: ClusterDeployment
+metadata:
+  name: {{ cluster_deployment_name }}
+  namespace: {{ assisted_namespace }}
+spec:
+  baseDomain: redhat.com
+  clusterName: {{ cluster_name }}
+  platform:
+    agentBareMetal:
+      apiVIP: ""
+      ingressVIP: ""
+      agentSelector:
+        matchLabels:
+          bla: aaa
+  provisioning:
+    imageSetRef:
+      name: '{{ cluster_image_set_name }}'
+    sshPrivateKeySecretRef:
+      name: {{ ssh_private_key_name }}
+    installStrategy:
+      agent:
+        sshPublicKey: '{{ ssh_public_key }}'
+        networking:
+          clusterNetwork:
+            - cidr: 10.128.0.0/14
+              hostPrefix: 23
+          serviceNetwork:
+            - 172.30.0.0/16
+        provisionRequirements:
+          controlPlaneAgents: 1
+  pullSecretRef:
+    name: {{ pull_secret_name }}
+EOCR
+
+echo "Running Ansible playbook to create kubernetes configs"
+ansible-playbook assisted-installer-crds-playbook.yaml
+
+echo "Assisted Installer pods"
+oc get pods -n assisted-installer
+
+echo "Metal3 pods"
+oc get pods -A | grep metal3
+
+echo "VM status"
+virsh list --all
+vm=""
+vm=$(virsh list --all | grep "shut off" | awk '{print $2}')
+echo "Offline VM is $vm"
+
+echo "Creating Assisted Installer Pull Secret"
+oc create secret generic ${ASSISTED_PULLSECRET_NAME} --from-file=.dockerconfigjson=${ASSISTED_PULLSECRET_JSON} --type=kubernetes.io/dockerconfigjson -n ${ASSISTED_NAMESPACE}
+
+echo "Creating Assisted Installer Private Key"
+oc create secret generic ${ASSISTED_PRIVATEKEY_NAME} --from-file=ssh-privatekey=/root/.ssh/id_rsa --type=kubernetes.io/ssh-auth -n ${ASSISTED_NAMESPACE}
+
+echo "Creating Assisted Installer Cluster Image Set"
+oc create -f clusterImageSet.yaml
+
+echo "Creating Assisted Installer SNO ClusterDeployment"
+oc create -f clusterDeployment.yaml
+
+echo "Creating Assisted Installer InfraEnv"
+oc create -f infraEnv.yaml
+
+echo "Creating new baremetal host using baremetal operator"
+oc create -f /root/dev-scripts/ocp/ostest/extra_host_manifests.yaml
+oc label -f /root/dev-scripts/ocp/ostest/extra_host_manifests.yaml "infraenvs.agent-install.openshift.io=myinfraenv"
```

Comment:
shouldn't it be
```suggestion
oc label -f /root/dev-scripts/ocp/ostest/extra_host_manifests.yaml "infraenvs.agent-install.openshift.io=${ASSISTED_INFRAENV_NAME}"
```
?